project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -70,7 +71,7 @@ public static double score(Restaurant restaurant, JSONObject preferences,
     if (hasRating) {
       score += calculateRatingScore(restaurantRating, restaurant.getNumRatings(), statistics);
     }
-    double percentMatch = score / maxPoints;
+    double percentMatch = score / maxPoints <= 1 ? score / maxPoints : 1.0;
```

Comment:
nit: I think Math.min(score/maxPoints, 1.0) would be clearer than the ternary here